apps/server/src/modules/deletion/api/uc/deletion-request.uc.ts

```diff
@@ -65,53 +65,45 @@ export class DeletionRequestUc implements IEventHandler<DataDeletedEvent> {
 
 	async createDeletionRequest(deletionRequest: DeletionRequestBodyProps): Promise<DeletionRequestResponse> {
 		this.logger.debug({ action: 'createDeletionRequest', deletionRequest });
+		const minutes =
+			deletionRequest.deleteAfterMinutes ?? this.configService.get<number>('ADMIN_API__DELETION_DELETE_AFTER_MINUTES');
```

Comment:
maybe setting 
`const defaultMinutes = this.configService.get<number>('ADMIN_API__DELETION_DELETE_AFTER_MINUTES')` before this line for readability?